Makefile

```diff
@@ -148,7 +148,7 @@ thirdparty-fast:
 	    if [ "$$new_zetasql_version" != "$(ZETASQL_VERSION)" ] ; then \
 		echo "[deps]: thirdparty up-to-date. reinstall zetasql from $(ZETASQL_VERSION) to $$new_zetasql_version"; \
 		$(MAKE) thirdparty-configure; \
-		$(CMAKE_PRG) --build $(THIRD_PARTY_BUILD_DIR) --target zetasql; \
+		$(CMAKE_PRG) --build $(THIRD_PARTY_BUILD_DIR) -j $(NPROC) --target zetasql; \
```

Comment:
revert here as well

Reply:
already done https://github.com/4paradigm/OpenMLDB/pull/3220/commits/631f6eaddbc4d744cfe03b0405fa6aa921d9c77f, but seems we can't see it